extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/common/DruidKubernetesClientTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.k8s.overlord.common;
+
+import io.fabric8.kubernetes.client.Client;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+
+class DruidKubernetesClientTest
+{
+
+  @Test
+  void makingCodeCoverageHappy()
```

Comment:
Hmm. This is a bit of a bummer. Is there no way to avoid this?

Reply:
So I had tests for a class, then i upgraded a client library and had to change some things.  but the functions were all tested before.  Now i would expect that if tests pass I would be good, since it was already tested before.  But turns out if you have to make any changes to make the new library apis work, then code coverage complains.  It is unfortunate here, because I didn't change any behavior.

Reply:
aren't there any tests covering that path?

Reply:
i added a getter to a class, the code coverage tool complained that i added new code without a test.  So I added a test for the getter.  That was new code, so i guess its okay to add a test if that's what druid wants.  Although I feel tests like this are not really that useful and make the whole suite slow in general.